website/versions.json

```diff
@@ -1,4 +1,5 @@
 [
+  "18.4.0",
```

Comment:
Not sure if this actually matters (but it might depending on how it's looked up) but I think this should be "v18.4.0"

Reply:
a good catch